index.html

```diff
@@ -148,17 +257,19 @@ <h2 class="content-text">About
                     <li class="separator"></li>
                   </ul>
             </div>
-            
+            </div>
+        </div>
         </div>
         <footer> 
             <section class="contact-me">
-                <div class="content">
+                <div class="contact-me-1">
```

Comment:
Please Notice the font style and font color in the contact me section and kindly fix it 🤗
Maybe add some padding in the input fields to make it resemble the Figma design 😃

Your design:

![image](https://user-images.githubusercontent.com/73714615/188266305-1f16e4c7-d873-42a7-8f82-314a543f11f5.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188266316-94404b04-b394-4e37-ac2a-a64c82c933e1.png)

Reply:
Good job following the advice of the previous reviewer